es/ch02.md

```diff
@@ -1,90 +1,99 @@
 
 **(1) Nombre del monedero:** Asigna un nombre a tu monedero (en este ejemplo se usa `testname`).    
 **(2) Semillas mnemónicas:** Pega tu semilla formada por 25 (o 24) palabras.    
 **(3) Altura de restauración (opcional):** Si conoces la altura de bloque en la que se creó tu monedero, puedes especificarla aquí para no tener que escanear toda la cadena de bloques en busca de tus fondos. Por ejemplo, si tu primera transacción se incluyó en el bloque `1350000`, puedes usar una altura ligeramente más baja (por ejemplo,` 1330000`) para que la monedero comience a escanear desde allí, ahorrando tiempo.    
-**(4) Ubicación de la cartera:** Selecciona la carpeta de destino del monedero.
-
-### Restaurando desde claves
+**(4) Ubicación del monedero:** Selecciona la carpeta de destino del monedero.
 
-La restauración desde las claves es bastante sencilla y puede ser extremadamente útil, especialmente si mueves tu monedero desde un servicio en línea como MyMonero. 
+Una guía detallada está disponible en getmonero.org: ['Cómo restaurar tu cuenta'](https://getmonero.org/resources/user-guides/restore_account.html).
 
-![restore from key](media/wizard_7-restore-keys.png)
+### Restaurando desde claves
+![Restaurar monedero desde claves.](media/wizard_7-restore-keys.png)
 
-Necesitas facilitar la siguiente información:
+La restauración desde las claves es bastante sencilla y puede ser extremadamente útil, especialmente si mueves tu monedero desde un servicio en línea como MyMonero. Necesitas facilitar la siguiente información:
 
 **(1) Nombre del monedero:** Asigna un nombre a tu monedero (en este ejemplo se usa `testname`).    
 **(2) Dirección de la cuenta:** La dirección de la monedero que estás recuperando.    
 **(3) Clave de vista:** Tu clave de vista privada (necesaria para poder verificar tus fondos).    
 **(4) Clave de gasto:** Tu clave de gasto privado (necesaria para gastar tus fondos).    
 **(5) Altura de restauración (opcional):** Si conoces la altura de bloque en la que se creó tu monedero, puedes especificarla aquí para no tener que escanear toda la cadena de bloques en busca de tus fondos. Por ejemplo, si tu primera transacción se incluyó en el bloque `1350000`, puedes usar una altura ligeramente más baja (por ejemplo,` 1330000`) para que la monedero comience a escanear desde allí, ahorrando tiempo.    
-**(6) Ubicación de la cartera:** Seleccione la carpeta de destino de la cartera.    
+**(6) Ubicación del monedero:** Seleccione la carpeta de destino de la cartera.    
 
 Cuando todo esté listo, haz clic en la flecha derecha y luego en el botón `Usar Monero`.
-En getmonero.org se pone a disposición de los usuarios una guía más avanzada: [[EN] Restoring wallet from keys](https://getmonero.org/resources/user-guides/restore_from_keys.html).
+En getmonero.org se pone a disposición de los usuarios una guía más avanzada: [Restaurar monedero desde claves](https://getmonero.org/resources/user-guides/restore_from_keys.html).
```

Comment:
If you remove the `[EN]` tag, here. I suggest to link to the spanish version of the userguide on the website

Reply:
Yes, thanks, I didn't know the link was different :)